inference-engine/include/ie_blob.h

```diff
@@ -807,7 +807,7 @@ class TBlob : public MemoryBlob {
     }
 };
 
-#if defined(__clang__) && !defined(__SYCL_COMPILER_VERSION)
+#if defined(ANDROID)
```

Comment:
If I remember right we also uses these macros for macOS.

Reply:
Not sure because they were introduced for Android only and related RTTI issue.